### ISSUE TITLE
[Keyboard-Layout] Symbols not typable: Brackets, braces, etc.

### ISSUE THREAD

**Issue by NullPlane** [SpartanJ/ecode#119]
#### On Windows:

Using **_german layout_** here and trying to code results in brackets, braces, etc. being unable to be typed using `CTRL+ALT+{Number Key}`, but only appear using the infamous "`AltGr`" (Right-Alt) key - which is _usually_ meant to do exactly the same as `CTRL+ALT+anything`. 

Checked if Numpad-Toggle changes anything (this often times causes similar problems) but it didn't change anything.

Is this a feature that's not yet been implemented or a bug or is this related to any specific cause in my system? 
I'd just found your Editor and thought about making it my new main for atm cuz nice work ngl. While typing quickly this literally makes quick working impossible.

ty in advance

---

```
version: Win11-23H3-22631.3085-Home
keyboard-interface: USB
driver: HID-Keyboard (win default, up to date)
win-layout-type: German (Germany) - QWERTZ
```

**Comment by SpartanJ** [comment]
I just [pushed a fix](https://github.com/SpartanJ/eepp/commit/30051cf6b4644d9a9b1e5bc8148b4c5f1d0290fe).
Here's is a patched build: [ecode-windows-0.5.1-x86_64.zip](https://github.com/SpartanJ/ecode/files/14127852/ecode-windows-0.5.1-x86_64.zip). This is a preview of the next release so it can be a little bit unstable. But feel free to use it. If you find it unstable please let me know and disable the Git plugin since that's basically where the big changes are.


Thanks for reporting it!

**Comment by NullPlane** [comment]
That went quick tho.

Using this new build, sadly it does still not work for some reason.

I cannot really say what it is tho.

Tried it using Windows's osk directly but it yields the same results:

![bug1](https://github.com/SpartanJ/ecode/assets/111517889/6df08cf4-6c47-42fb-8eb6-607e36858563)


> I just [pushed a fix](https://github.com/SpartanJ/eepp/commit/30051cf6b4644d9a9b1e5bc8148b4c5f1d0290fe). Here's is a patched build: [ecode-windows-0.5.1-x86_64.zip](https://github.com/SpartanJ/ecode/files/14127852/ecode-windows-0.5.1-x86_64.zip). This is a preview of the next release so it can be a little bit unstable. But feel free to use it. If you find it unstable please let me know and disable the Git plugin since that's basically where the big changes are.
> 
> Thanks for reporting it!

**Comment by SpartanJ** [comment]
Ups!! Sorry, I tested it on a Windows VM and copied the fix incorrectly. This one should be OK:
[ecode-windows-0.5.1-x86_64.zip](https://github.com/SpartanJ/ecode/files/14129185/ecode-windows-0.5.1-x86_64.zip)

**Comment by NullPlane** [comment]
### Oh ya baby now we're talking.

_Works flawlessly. Thanks for the immediate response and fix incl. precompile._  
_Hereby closed at [30051cf](https://github.com/SpartanJ/eepp/commit/30051cf6b4644d9a9b1e5bc8148b4c5f1d0290fe), [8cbc580](https://github.com/SpartanJ/eepp/commit/8cbc5806f03f8c5f6d18167b4ca4ec9e9c31c82b)._

**Comment by SpartanJ** [comment]
I'll reopen it until 0.5.1 is released. I'm glad it worked!
